bftengine/src/communication/TlsTCPCommunication.cpp

```diff
@@ -750,13 +728,6 @@ class AsyncTlsConnection : public enable_shared_from_this<AsyncTlsConnection> {
   }
 
   void write_async(const char *data, uint32_t length) {
-    // async_write(socket,
```

Comment:
Does this mean that write_async is not actually async?

Reply:
Yes, I need to change the name. The write operation is sync since we want to have a confidence that the message has been sent before sending another one.

Reply:
Yes, we want to have more confidence that the message has been sent before sending the next one. Will change the method name.